tests/acceptance/test_mode_transitions.py

```diff
@@ -43,7 +42,7 @@ def test_antenna_trans_to_stow(cm_manager_connected_to_antnn):
     """Test the stow function"""
     cm_manager_connected_to_antnn.set_stow_mode()
     # The antenna takes roughly 30 sec to switch modes
-    time.sleep(33)
+    time.sleep(45)
```

Comment:
In other pull requests, the antenna transition time is 30-33 sec and the tests pass. Is there something that has changed on the simulator regarding this pull request?